### PR TITLE
[FLINK-6709] [gelly] Activate strict checkstyle for flink-gellies

### PR DESCRIPTION
**Pull request by greghogan**

### CODE REVIEW

File: flink-libraries/flink-gelly-examples/src/main/java/org/apache/flink/graph/examples/data/TriangleCountData.java [hunk 1]
```diff
@@ -33,7 +33,15 @@
  */
 public class TriangleCountData {
 
-	public static final String EDGES = "1	2\n"+"1	3\n"+"2	3\n"+"2	6\n"+"3	4\n"+"3	5\n"+"3	6\n"+"4	5\n"+"6	7\n";
```
Comment (zentol) [inline flink-libraries/flink-gelly-examples/src/main/java/org/apache/flink/graph/examples/data/TriangleCountData.java]: are we replacing spaces with tabs here?

Comment (greghogan) [inline flink-libraries/flink-gelly-examples/src/main/java/org/apache/flink/graph/examples/data/TriangleCountData.java]: No, there just happen to be 7 characters between tabs so they appear as spaces after the first.

Comment (zentol) [inline flink-libraries/flink-gelly-examples/src/main/java/org/apache/flink/graph/examples/data/TriangleCountData.java]: i see. Want me to include this in my next merge? along with `flink-fs-tests` and `flink-quickstart`.

Comment (greghogan) [inline flink-libraries/flink-gelly-examples/src/main/java/org/apache/flink/graph/examples/data/TriangleCountData.java]: Sure, that'd be great.

**Pull request closed without merging**